pom.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>com.patrickjones</groupId>
+	<artifactId>bank</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<packaging>jar</packaging>
+
+	<name>bank</name>
+	<description>First Bank of Patrick</description>
```

Comment:
This is the `pom.xml` file. It stands for "project object model." It tells Maven how to build our project. Shortly, we'll start adding dependencies to it as we build up the functionality of our bank.